notifier.rst

```diff
@@ -121,7 +122,7 @@ Service             Package                                DSN
 
 .. versionadded:: 6.4
 
-    The `Redlink`_ and `Brevo`_ integrations were introduced in Symfony 6.4.
+    The `Redlink`_, `Brevo`_ and `SmsSluzba`_ integrations were introduced in Symfony 6.4.
```

Comment:
you must target https://github.com/symfony/symfony-docs/tree/7.1 and swap here :)